src/main/java/org/simulator/io/CSVImporter.java

```diff
@@ -27,13 +27,7 @@
 import java.io.BufferedReader;
 import java.io.FileReader;
 import java.io.IOException;
-import java.util.AbstractList;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.LinkedList;
-import java.util.List;
-import java.util.Map;
-import java.util.Arrays;
+import java.util.*;
 import java.util.logging.Logger;
```

Comment:
It seems we are not consistent in the use of loggers. Here we are using the on-board logger from Java (which is good because it is always shipped directly with the Java distribution) but in other places we are using the log4j logger (which is also good because it comes with JSBML anyway and it has more features). We should decide which logger to use and stick to that one.

Reply:
@draeger, I think it is better to use the log4j logger as it comes from JSBML anyways.